### PR TITLE
Db camel case

### PR DESCRIPTION
**Pull request by LevoraShelby**
Made db column names camelcase and made route names hyphen-spaced.

Merging into `notification` since it's where I branched from. I branched from it since it was still active and was adding column references.

### CODE REVIEW

File: server/models/listProductModel.js [hunk 1]
```diff
@@ -3,27 +3,38 @@ const UserListModel = require("./userListModel");
...
+//   },
+//   onDelete: "CASCADE",
+// });
+// ProductModel.hasMany(ListProductModel, {
+//   foreignKey: {
+//     name: "productId",
+//     allowNull: false,
+//   },
+//   onDelete: "CASCADE",
+// });
```
Comment (Fralalala) [inline server/models/listProductModel.js]: Will this be transferred onto a different file? 

Or maybe this was just commented out for testing purposes?

Comment (LevoraShelby) [reply server/models/listProductModel.js]: oh yea, that needs to get removed! I'll get on that.

Comment (Fralalala) [reply server/models/listProductModel.js]: Arent the listId and productId used to find which products are in a list?

Comment (LevoraShelby) [reply server/models/listProductModel.js]: Yes. I'm using a different association method called `belongsToMany` to set up those columns.

**Review by sina-jamshidi — APPROVED** [review]
Thanks! Just resolve merge conflicts and also make sure you've tested it and things still work lol

**Pull request merged**